tests/common.py

```diff
@@ -85,7 +85,6 @@ def get_test_home_assistant():
     def run_loop():
         """Run event loop."""
         # pylint: disable=protected-access
-        loop._thread_ident = threading.get_ident()
```

Comment:
This one is still great to have because this is actually a standalone thread running the loop. And since the check does catch errors, it would be nice to still have.